jobs/rocoto/atmos_products.sh

```diff
@@ -15,13 +15,17 @@ if (( status != 0 )); then exit "${status}"; fi
 export job="atmos_products"
 export jobid="${job}.$$"
 
-# Negatation needs to be before the base
-fhr3_base="10#${FHR3}"
-export FORECAST_HOUR=$(( ${fhr3_base/10#-/-10#} ))
+# shellcheck disable=SC2153
+IFS=', ' read -r -a fhr_list <<< "${FHR_LIST}"
 
-###############################################################
-# Execute the JJOB
-###############################################################
-"${HOMEgfs}/jobs/JGLOBAL_ATMOS_PRODUCTS"
+export FORECAST_HOUR
+for FORECAST_HOUR in "${fhr_list[@]}"; do
+	###############################################################
+	# Execute the JJOB
+	###############################################################
+	"${HOMEgfs}/jobs/JGLOBAL_ATMOS_PRODUCTS"
+	status=$?
```

Comment:
I recommend moving `jobid` here and naming it to something like the suggestion below. Please see my comment above. 
```suggestion
   jobid="${job}_${FORECAST_HOUR}.$$"
	"${HOMEgfs}/jobs/JGLOBAL_ATMOS_PRODUCTS"
	status=$?
```

Reply:
excellent suggestion @EricSinsky-NOAA

Reply:
Added these for all the groupified jobs.